### ISSUE TITLE
unable to see path subbed playlists when looking to create a shortcut for a playlist. 

### ISSUE THREAD

**Issue by vajonam** [bartolomesoriano/skin.amber#32]
When using path substitution, I am not able to find any of my video playlists. However when display just "Playlists" I can navigate to them and execute them.

```

<substitute>
          <from>special://profile/playlists/video</from>
          <to>nfs://NFS_SERVER/NFS_PATH/playlists/video</to>
        </substitute>
```

I need to do this because on the NVIDIA Shield I cannot mount a NFS share and symlink to it like I have other other linux based clients. I can do a custom menu item but will nice to get this working. anyone else have this issue?

**Comment by bartolomesoriano** [comment]
Does this work in the default Estuary skin? This does not seem to be a skin issue.

**Comment by vajonam** [comment]
Yes, I am able to see then when I open the playlists folder. I think it might be an issue with https://forum.kodi.tv/showthread.php?tid=178294&pid=2546915#pid2546915 skin.shortcuts.

**Comment by bartolomesoriano** [comment]
Thanks.  Amber uses skin.shortcuts, but I am not the developer of that addon. 
To be clear, you cannot see your playlists when trying to add/change a menu/submenu item that you want to point to your playlist, correct?

**Comment by vajonam** [comment]
>  To be clear, you cannot see your playlists when trying to add/change a menu/submenu item that you want to point to your playlist, correct?

Yes exactly. Browsing the the "Playlist  >" nodes only shows the skin playlists and not that is in the subbed path. however when I add it manually. Everything works as expected from then on. 

```
        <shortcut>
                <defaultID>hdaudio</defaultID>
                <label>HD Audio</label>
                <label2>Video Playlist</label2>
                <icon>DefaultPlaylist.png</icon>
                <thumb />
                <action>ActivateWindow(Videos,special://profile/playlists/video/HD Audio.xsp,return)</action>
                </shortcut>
```

**Comment by vajonam** [comment]
Added a link here. https://github.com/BigNoid/script.skinshortcuts/issues/247
